forge/lib/tiler.py

```diff
@@ -245,6 +302,118 @@ def create(self):
         logger.info('It took %s to create %s tiles (%s were skipped)' % (
             str(datetime.timedelta(seconds=tend - self.t0)), tilecount.value, skipcount.value))
 
+    # Create AWS sqs queue with all the tiles to create
+    # based on current configuration as well as meta data
+    def createQueue(self):
+        queueName = self.tmsConfig.get('General', 'sqsqueue')
+        maxChunks = int(self.tmsConfig.get('General', 'maxChunks'))
+        self.t0 = time.time()
+        if len(queueName) <= 0:
+            logger.error('Missing queueName')
+            return
+        try:
+            sqs = getSQS()
+            q = sqs.get_queue(queueName)
+            if q is not None:
+                logger.error('Queue already exists. Can\'t overwrite existing queue. [%s]' % (queueName))
+                return
+
+            # queue with default message visiblity time of 3600. So each message is blocked
+            # for other users for 3600 seconds. Default would be 30 seconds. It's that high
+            # because each message contains maxChunks tiles
+            q = sqs.create_queue(queueName, visibility_timeout = visibility_timeout)
+            # Assure queue is kept for maximum of 14 weeks (aws limit). default would be 4 days.
+            sqs.set_queue_attribute(q, 'MessageRetentionPeriod', 1209600)
+        except Exception as e:
+            logger.error('Error during creation of queue' + str(e))
+            return
+
+        if q.count() > 0:
+            logger.error('Queue already contains messages. Use a different queue or delete this queue first')
+            return
+
+        logger.info('Queue ' + queueName + ' has been created')
+        tiles = Tiles(self.dbConfigFile, self.tmsConfig, self.t0)
+        nbTiles = self.numOfTiles()
+        try:
+            logger.info('Starting creation of SQS queue with approx. %s tiles)' % (nbTiles))
+            totalcount = 0
+            tcount = 0
+            messagecount = 0
+            msg = ''
+            for tile in tiles:
+                (bounds, tileXYZ, t0, dbConfigFile) = tile
+                if len(msg) != 0:
+                    msg += ','
+                msg += ('%s,%s,%s' % (str(tileXYZ[0]), str(tileXYZ[1]), str(tileXYZ[2])))
+                tcount += 1
+                if tcount >= maxChunks:
+                    messagecount += 1
+                    writeSQSMessage(q, msg)
+                    tcount = 0
+                    msg = ''
+                totalcount = totalcount + 1
+            if len(msg) > 0:
+                messagecount += 1
```

Comment:
no need to count messages because the number of messages is ` int(math.ceil(float(nbTiles)/float(maxChunks)))`